libs/api-provider-environment/src/webb-context/webb-context.ts

```diff
@@ -109,31 +112,29 @@ export const WebbContext = React.createContext<WebbContextState<unknown>>({
   loading: true,
   activeAccount: null,
   noteManager: null,
-  loginNoteAccount(key: string, walletAddress: string) {
```

Comment:
CC @AtelyPham @vutuanlinh2k2, please do give this all a review @devpavan04 why updating all these function signatures?